android/support/build.gradle

```diff
@@ -0,0 +1,23 @@
+repositories {
+    jcenter()
+    google()
+}
+
+apply plugin: "com.android.library"
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+    }
```

Comment:
Is there any way to keep these in a central place so we don't have to manually sync with `android/app/build.gradle`?

Reply:
Good point, thanks.